reactos/drivers/usb/usbhub/fdo.c

```diff
@@ -2044,7 +2106,6 @@ USBHUB_FdoHandlePnp(
         }
     }
 
-    Irp->IoStatus.Information = Information;
     Irp->IoStatus.Status = Status;
     IoCompleteRequest(Irp, IO_NO_INCREMENT);
```

Comment:
This seems correct. Seems like the function could be simpler though if you changed this "default" to call ForwardIrpAndForget, and just have START_DEVICE be the exception